% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
]{article}

\usepackage{amsmath,amssymb}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else  
    % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage[lmargin=20mm,rmargin=20mm]{geometry}
\setlength{\emergencystretch}{3em} % prevent overfull lines
\setcounter{secnumdepth}{5}
% Make \paragraph and \subparagraph free-standing
\makeatletter
\ifx\paragraph\undefined\else
  \let\oldparagraph\paragraph
  \renewcommand{\paragraph}{
    \@ifstar
      \xxxParagraphStar
      \xxxParagraphNoStar
  }
  \newcommand{\xxxParagraphStar}[1]{\oldparagraph*{#1}\mbox{}}
  \newcommand{\xxxParagraphNoStar}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
  \let\oldsubparagraph\subparagraph
  \renewcommand{\subparagraph}{
    \@ifstar
      \xxxSubParagraphStar
      \xxxSubParagraphNoStar
  }
  \newcommand{\xxxSubParagraphStar}[1]{\oldsubparagraph*{#1}\mbox{}}
  \newcommand{\xxxSubParagraphNoStar}[1]{\oldsubparagraph{#1}\mbox{}}
\fi
\makeatother


\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother

\usepackage{lineno}\linenumbers
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\AtBeginDocument{%
\ifdefined\contentsname
  \renewcommand*\contentsname{Table of contents}
\else
  \newcommand\contentsname{Table of contents}
\fi
\ifdefined\listfigurename
  \renewcommand*\listfigurename{List of Figures}
\else
  \newcommand\listfigurename{List of Figures}
\fi
\ifdefined\listtablename
  \renewcommand*\listtablename{List of Tables}
\else
  \newcommand\listtablename{List of Tables}
\fi
\ifdefined\figurename
  \renewcommand*\figurename{Figure}
\else
  \newcommand\figurename{Figure}
\fi
\ifdefined\tablename
  \renewcommand*\tablename{Table}
\else
  \newcommand\tablename{Table}
\fi
}
\@ifpackageloaded{float}{}{\usepackage{float}}
\floatstyle{ruled}
\@ifundefined{c@chapter}{\newfloat{codelisting}{h}{lop}}{\newfloat{codelisting}{h}{lop}[chapter]}
\floatname{codelisting}{Listing}
\newcommand*\listoflistings{\listof{codelisting}{List of Listings}}
\makeatother
\makeatletter
\makeatother
\makeatletter
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\@ifpackageloaded{subcaption}{}{\usepackage{subcaption}}
\makeatother

\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\usepackage[style=authoryear,]{biblatex}
\addbibresource{../library.bib}
\usepackage{bookmark}

\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same} % disable monospaced font for URLs
\hypersetup{
  pdftitle={Discussion},
  colorlinks=true,
  linkcolor={blue},
  filecolor={Maroon},
  citecolor={Blue},
  urlcolor={Blue},
  pdfcreator={LaTeX via pandoc}}


\title{Discussion}
\author{}
\date{}

\begin{document}
\maketitle


\section{Introduction}\label{introduction}

Hastened genetic progress in potato is dependent on the continued
development and application of quantitative genetics methodology in
potato breeding. Like many vegetable crops, the application of
statistical genetic theory has only received significant attention the
past two decades in potato. Before this, complex trait improvement in
conventional potato breeding was reliant on phenotypic selection and
marked by several inefficiencies including underpowered field
evaluations and long breeding cycles \autocite{Bradshaw2017}. With the
continued application of technologies that have had positive success in
other field crops, increased genetic improvement in potato is possible.
These include both methods such as variance component estimation,
genomic prediction, selection index theory as well as the importing of
other breeding schemas like recurrent selection or line breeding.

In this thesis, I examine the conversion of clonal tetraploid potato to
a diploid hybrid crop through the lens of biometrical methods. To our
knowledge, this thesis is the first to interrogate the genetic
architecture of a large F1 crossing block in diploid hybrid potato.
Moreover, these past experimental chapters also took the opportunity to
assess the adequacy of multiple statistical methods in the estimation
and prediction of genetic value in this novel crop. I first examined the
genetical properties of the F1 hybrid seedling trials including genetic
correlations between multiple tuber yield components (chapter 2;
\textcite{Adams2022}). I also utilized multiple forms of genetic
information to assess the family history of the inbred parental lines.
This included the use of pedigree and molecular marker information
(chapter 3; \textcite{Adams2023}) as well as identity by descent
profiles (chapter 5) to evaluate the population structure of these
parental lines. After having estimated these genetic parameters, I
tested multiple predictive modelling methodologies testing the potential
benefits of predicting dominance effects (chapter 3;
\textcite{Adams2023}), the utility of marker assisted selection for
complex traits (chapter 4), and various multiallelic parameterisations
for genome-wide regression (chapter 5). These topics cover an important
range of topics for modern selection methods in a potato breeding
program.

Here I provide a synthesis of the previous four experimental chapters
with two major aims. I first wish to identify the most important
findings of this research, and second, outline the direct implications
of our research on potato breeding more generally. I then conclude this
thesis with reflections on future research.

\subsection{Biometrical analysis of field
trials}\label{biometrical-analysis-of-field-trials}

A core activity in breeding programs is thoe regular evaluation of new
parental combinations to screen for genetically improved offspring.
Identification of suitable and informative trialling locations ensures
an unbiased and representative assessment of a candidate's performance.
For this reason, within-trial and multi-environmental trial design is so
heavily emphasized in modern crop literature for sufficient statistical
power and optimal resource allocation \autocite{Schoemaker2024}. This
permits proper extraction of spatial and other non-genetic components in
a field trial and unbiased evaluation of genotype performance. From this
successful spatial analysis, it was then possible to measure the genetic
parameters of interest whether it be comparison of genotype means, or
the estimation of genetic correlations between traits.

In conventional potato breeding, formal designed trials are reserved for
the 3\textsuperscript{rd} or 4\textsuperscript{th} clonal generations
when enough clonal propagule can guarantee sufficient replication
\autocite{Gopal2015,Paget2017}. This is in contrast to other field crops
where true seed is the predominant source of germplasm and allows for
suitable replication after the initial cross or selfing
\autocite{Zystro2018}. An overlooked property of hybrid breeding in
potato is the potential of leveraging true seed and the ability to
conduct trialing on multi-plant plots over traditional tuber-sown hills
\autocite{Davies1974,Pallais1991}. The latter tend to be underpowered
with too low of precision for early selection making true potato seed a
powerful resource in breeding germplasm evaluation
\autocite{Storck2011,Stockem2021}. This is not without negative
consequences, chief among them, being the poor correlation in multiple
tuber traits between the seedling and clonal generations
\autocite{Anderson1981,Ortiz1997,Gu2024}. This is addressed in full in
Section~\ref{sec-implications}, but suffice it to say that the field
evaluation of hybrid potato via seedling transplants provides
opportunities for explicit field design with sufficiently high precision
for multiple tuber phenotypes \autocite{VanDijk2021a}.

In \textbf{chapter 2}, such an exercise was conducted in the form of a
multi-trial analysis on several hundred F1 hybrids. A penalized splines
based procedure was used for spatially de-trending our hybrid
observations for three tuber phenotypes. These de-trended phenotypes
were then placed in a multivariate linear mixed model for the estimation
of the genetic (co-)variances for the GCA, SCA, and several interactive
effects. In \textbf{chapter 3}, this was followed up with a traditional
linear mixed modelling procedure where multiple row column designs with
various residual structures were tested on an augmented dataset
\autocite{Gilmour1997}. The BLUEs and standard errors from these models
where then used in a full genomic model where genomically estimated
BLUPs were evaluated in a cross-validation schema. While the spatial
modelling procedures from \textbf{chapter 2} and \textbf{chapter 3} were
derived from two different frameworks, little to no difference was found
between the use of semi-parametric Spline functions and structuring our
residual variance. Previous studies have also suggested these approaches
as equivalent with exception of the ease of a single model-fitting over
the testing of multiple linear mixed models \autocite{Velazco2017}. Both
modelling procedures from these chapters follow the general form of a
two-stage mixed model analysis where trials are independently analyzed
followed by an combined analysis on the genotypic means (most commonly)
together with weights on those genotypic observations
\autocite{Piepho2012}. Two-stage approaches are understood to be
generally less accurate than than their single stage counterparts
especially with greater model complexity \autocite{Gogel2018}. All this
being said, single-stage approaches tend to be unwieldy in their
specification and computation and the consequences of a two-stage
approach are unlikely to change any inferential conclusions from these
two chapters \autocite{Damesa2017}.

A core concept in hybrid breeding is the ability to assess the potential
of hybrid crosses on the basis of parental value. In the case for
phenotypic selection, general combining abilities tend to be the primary
mode which communicates a line's value \autocite{Bertan2007}. In
\textbf{chapter 2}, GCAs were generated for 400 parental lines in
multiple tuber traits and found that the GCAs sufficiently captured
between 59 and 71 per cent of the observed phenotypic variation in the
F1 screening trials. The use of GCA for selection is not unique to
hybrid crops with many examples of their use in tetraploid potato in
many modern applications as well \autocite{Neele1991,Bradshaw2003}. This
is despite many tetraploid studies reporting a large proportion of
non-additive genetic effects controlling tuber qualities like tuber
size, tuber yield, and marketable yield
\autocite{Plaisted1962,Tai1976,Killick1977}.

From \textbf{chapter 2}, some insight around the genetic architecture of
yield components was gleaned in the form of genetic correlations. This
was not just expressed in terms of a single genetic component over the
hybrids, but explicitly in terms of the additive and non-additive
genetic covariance (\(\Sigma_G = 2 \cdot \Sigma_{gca} + \Sigma_{sca}\)).
This is worth re-emphasis. Potato has many traits are relevant for
selection; many of them being complex traits with low heritabilities and
interdependent relationships \autocite{Gebhardt2013}. Because of this,
it is critical for breeders to better utilize multivariate methods in
genetic improvement in potato. In addition, hybrid breeding schemas are
also augmented by multivariate methods whether based upon phenotype
alone or also for multivariate genomic prediction applications
\autocite{Wang2017}.

\subsection{Genomic prediction in hybrid
potato}\label{genomic-prediction-in-hybrid-potato}

Genomic prediction has emerged as a cornerstone of modern crop breeding,
offering unprecedented precision in estimating genetic merit by
leveraging genome-wide marker data. By modelling molecular markers with
phenotypic records, genomic prediction enables breeders to identify
superior parental combinations and hybrid crosses with greater accuracy
and efficiency than traditional pedigree-based methods. This section
explores the application, advantages, and limitations of genomic
prediction in hybrid potato breeding, drawing on insights from our
experimental chapters to explore its role in accelerating genetic gain
in potato.

Multiple chapters in this thesis addressed genomic prediction
applications in hybrid potato. In \textbf{chapter 3}, the genetic models
partitioned into GCA and SCA from \textbf{chapter 2} were built upon
with incorporated molecular marker information to structure the each
genetic component. Multiple genomic models were then tested in a
predictive application testing a simple and full genetic model (GCA, and
GCA+SCA, respectively) for the same tuber variates from \textbf{chapter
2} along with tuber dry matter content. Testing the predictive model
performance between each model showed no additional benefit through the
addition of the SCA component in the model. Contrasting the SCA variance
relative to the genetic residual suggested that there were other genetic
effects that were not captured in either of the other genetic effects,
most notably in total tuber number and dry matter content. Through this
modelling schema, it was confirmed that genomic prediction solely on the
basis of GCA sufficed in the estimation of a hybrid cross genetic for
all tuber variates studied.

These prediction models were extended further in \textbf{chapter 5} by
examining several other statistical paradigms and assessing any benefits
to multiallelic marker information. Specific attention was taken to
examining the utility of identity-by-descent (IBD) information derived
from deep pedigree information linking ancestral founders to the
parent's of hybrids along with multiallelic identity-by-state (IBS)
information. Both types of marker information were compared with
conventional biallelic SNPs using traditional shrinkage-based models
along with more complicated kernel prediction. For all tuber variates,
the SNP based prediction models were superior than their multiallelic
counterparts. Marked differences in prediction accuracy were especially
observed between the SNP and IBD models. Similarly, little differences
between the different modelling methods were found with exception that
the Gaussian kernel tended to maximize prediction accuracy regardless of
trait and markerset. These results would suggest that in the context of
genomic prediction, simpler marker parameterizations tend to yield more
consistent prediction outcomes. Despite these results, multi-allelic
marker information are likely to continue to have a prominent role in
inferential applications such as multi-parental population (MPP) mapping
\autocite{Li2021a}.

\subsection{Methods of selection}\label{methods-of-selection}

Modern breeding programs are met with unbounding choice with regard to
different technologies. This is especially true for molecular marker and
marker-based methods of selection. Marker-assisted selection (MAS) and
other marker-based techniques have revolutionized modern breeding
programs by enabling precise selection of desirable traits at early
stages of development. In the context of hybrid potato breeding, these
methods leverage molecular markers to identify and select parental lines
with favorable alleles, thereby accelerating genetic gain.

The topic of selection strategy in hybrid potato was inspected in
\textbf{chapter 4}. The exercise comprised of comparing several
molecular marker-assisted strategies based upon their relative
efficiency which primarily depended on the prediction accuracy of the
underlying model used. The primary aim was to asses whether
marker-assisted selection could be as efficient as the genome-wide
regression models utilized in \textbf{chapter 3}. Three strategies were
considered: a marker-assisted selection strategy (\texttt{MA}),
genome-wide prediction (\texttt{GW}), and a single-SNP control strategy
(\texttt{PC}). Each of these were used to predict the value of an inbred
parent. Based upon a forward-selection procedure, 33 unique QTL were
found across three traits (with total tuber number being excluded).
These QTL were able to pick up between 54 and 56 per cent of the total
genetic variation and were the basis for prediction in the models used
in \texttt{MA}. When examining each strategy's selection accuracy, the
\texttt{GW} strategy was evidently superior for all three traits with
the relative efficiency of \texttt{MA} being between 0.89 to 0.95
relative to \texttt{GW}. Because these strategies have different costs
associated with them, primarily related to genotyping cost, these were
also considered. When the genotyping costs were integrated into the
relative efficiency, the \texttt{MA} appeared more favourable.

\section{Wider Implications}\label{wider-implications}

\subsection{Hybrid Breeding Schema}\label{hybrid-breeding-schema}

Two important questions broached in \textbf{chapter's 2 \& 3} dealt with
the nature of gene action in hybrid potato and what population and
breeding strategy should be leveraged to effectively breed for complex
trait improvement. Both of these questions impact the future of what
\emph{kind} of hybrid crop potato might be. While the scope of this
thesis is limited by the relatively narrow genetic background sampled to
initialize these inbred populations \autocite{Lindhout2018}, these
results can still be taken into consideration to inform strategy for
hybrid potato breeding.

One of the major findings \textbf{chapter 2} was a distinct lack of SCA
variance found among the panel of 806 hybrids. This was further
confirmed in \textbf{chapter 3} in a genomic model were the SCA variance
was smaller than both the GCA and genetic residual variance components.
As remarked in these chapters, this would indicate a lack of
non-additive gene action at work among our panel of F1 hybrids. This
could be characteristic of very little population structure among the
inbreds (as confirmed by the population-based analyses in
\textbf{chapter 3}), however, this explanation is not wholly
satisfactory. It has been observed in other hybrid crops that SCA
variance tends to be more present in complex trait architecture where
heterotic pools are not genetically distinct \autocite{Zhao2015b}.
Contrary to this, my results follow the more general pattern seen in
other hybrid crops where the dominance variance appears to be nothing
more than a genetic residual variance with most of the genetic variation
being allocated to the additive variance \autocite{Bernardo2016}. As a
complimentary argument, there are also many examples of higher-order
genic interactions which manifest in statistical models as additive
genetic variance \autocite{Hill2008,Monnahan2015}. This too could be
contributing to what was observed earlier in both phenotypic analyses
and subsequent predictive modelling. It is worth acknowledging that this
whole discussion hinges on the assumption of a direct relationship of
estimated statistical and genetic parameters derived therefrom
\autocite{Campos2015}. It is out of scope to defend this premise of
applied genetics here, but there are many examples of such genetic
parameters being accurate enough to guide decision-making in breeding
applications \autocite{Walsh2004,Daetwyler2013}. In other words,
``\emph{While these {[}assumptions{]} are formally unrealistic, methods
work.}'' \autocite{Hill2010}.

Perhaps most pertinent to the discussion of hybrid breeding is with
regard to pool structure and the expediency of multiple pools in potato.
The development of pools in crops like maize or sorghum was a relatively
unguided process where complementation between distinct genetic groups
was first observed often with pedigree-breeding, and then further
developed with more formal methods of population improvement
\autocite{Duvick2004}. Hybrid crops are primarily the product of a
multi-pool system, however, this is dependent upon multiple conditions
related both to complex trait improvement and the additional costs of
logistics in multi-pool breeding. Multiple simulation studies have
examined the topic of heterotic pool development and are worth
consideration for potato. Simulation of multiple tetraploid and diploid
breeding program designs found that a two pool strategy based around
GCA-based selection was effective in an inbred-hybrid program. However,
this was dependent on the trait's degree of dominance in addition to
hybrid schemas being more capital intensive \autocite{Labroo2023}. The
authors also remarked on the sensitivity of this strategy not only to
cycle length (especially if reliant on phenotypic-based screening), but
also the time required to generate fully inbred pools.

The question of how pools should be generated is also open for
deliberation. In terms of quantitative trait improvement, multiple
\emph{splitting} strategies have been suggested to be equivalent
including even random pool assignment, at least among selfing crops
\autocite{Cowling2020}. A more important consideration than genetic
differentiation among pools in potato will be the market segment
requirements. These will strongly govern pool development around trait
targets which are widely discordant. For example, it would be likely
cumbersome to produce parental lines for ware and quick service markets
from the same pool based upon the diverging product profiles of each
\autocite{Keijbets2008}. Rather than seeing heterotic pooling as some
necessity, it is instead the working out of a comprehensive strategy
dependent on the robust production of inbred lines, strong fertility
characteristics, and a reliable cytoplasmic male sterility mechanism
\autocite{McGrath2018,Eggers2024}. These are the core factors which are
decisive to the choice of single, two, or other multi-pool breeding
schemas. The topic of fertility is considered further in
Section~\ref{sec-future}.

\subsection{Multivariate applications for potato
breeders}\label{sec-implications}

Multivariate methods have already proven to benefit animal and crop
breeding by enabling simultaneous improvement of multiple traits while
delineating their genetic, environmental, and residual correlations.
These approaches are particularly valuable in potato breeding, where
traits such as tuber yield, tuber quality parameters, and disease
resistance often exhibit complex interdependencies. By integrating
genetic covariances, breeders can design selection indices that maximize
genetic gain across traits, thereby optimizing resource allocation and
accelerating progress toward breeding objectives. While these methods
have been widely adopted in other field crops, their application in
potato remains under-explored.

To lend some credence to multivariate selection in potato, let us
consider what multivariate selection would involve using the trait
genetic covariances reported from \textbf{chapter 2} (Figure
\ref{fig:gca-coef-full-pairs}) including estimated GCA covariance
matrices together with their full phenotypic variance matrix
(\(\mathrm P\)). Assuming the GCA variances estimated here would be
roughly equivalent to those estimated from a test cross schema in a
hybrid breeding program (\(\mathrm {G} \approx \mathrm {V}_{gca}\)), the
selection response on inbred parents could be estimated for future
inbred development using the multivariate breeder's equation as
expressed by \textcite{Lande1983}. If we performed truncation selection
on average tuber volume (\(s_{tv}~=~2~cm^3\)) with no direct selection
on total tuber yield and tuber number constant
(\(s_{ty}~=~0~Tonnes~\cdot~Ha^{-1}\) \(s_{tn}~=~0~Tubers\)), and
conducted inter-mating between selected candidates, then the expected
selection response (\(\mathrm R\)) next test cross cycle could be
estimated as:

\[ \mathrm {R = G \cdot P^{-1} \cdot S}\]
\[ \mathrm {R} = \begin{bmatrix}11.54 & 7.27 & 4.95 \\ 7.27 & 11.13 & 67.2 \\ 4.95 & 67.2 & 629.51\end{bmatrix}\cdot\begin{bmatrix}0.17 & -0.22 & 0.02 \\ -0.22 & 0.34 & -0.03 \\ 0.02 & -0.03 & 0\end{bmatrix} \cdot \begin{bmatrix} 2 \\ 0 \\ 0 \end{bmatrix} \]
\[\mathrm {R} = \begin{bmatrix}1.09 \\ 0.64 \\ 0.07\end{bmatrix}\]

This would indicate an increase of 1.1 \(cm^3\) in tuber volume next
test cross cycle with a minor increase in total tuber yield and little
change in tuber number. Note that not only has this increased our
selection response when used over the univariate alternative (0.9
\(= h^2 s\)), but we are also able to evaluate the impact of indirect
selection among the other tuber variates. This is invaluable both in
forecasting and breeding strategy development. More attention is needed
here to make these methods more practical to wield as well as scalable
in applied settings. With regard to practicality, selection indices are
one of the primary tools used to reduce a breeder's multi-dimensional
trait space into into a singular index with a variety of methods
proposed \autocite{Kempthorne1959,Bulmer1981}. Speaking to
implementability, the most challenging step in this process is the
estimation of the trait genetic covariance. This tends to become quite
unwieldy using traditional linear mixed modeling methods in
higher-dimensional spaces making other estimation methodology more
attractive \autocite{Blows2009,Runcie2013}.

This same exercise has utility in other facets of potato breeding. One
recurrent hurdle in potato evaluation is the lack of genetic correlation
between the initial seedling stages and subsequent clonal generations
\autocite{Maris1988}. These differences are not restricted to tubers
from these generations but are present earlier with major differences in
plant architecture suggesting altered sink-source dynamics between
generations \autocite{Gu2025}. This is true both for phenotypic
evaluation and genetic parameter estimation and all this significantly
constrains early selection efficiency within potato breeding programs
\autocite{Davies1974,Gopal1998}. Multivariate estimation of a
candidate's \emph{genetic value} (relevant in clonal programs) or
\emph{breeding value} (relevant for both clonal and hybrid breeding
schemas) jointly across multiple generations could be a valuable
extension of multivariate selection. The estimation of these
inter-generational covariance structures together with a with
traditional multivariate methods could potentially augment forecasting
the potential of a varietal candidate earlier in selection cycles.

\subsection{Successful genomic prediction in hybrid
potato}\label{successful-genomic-prediction-in-hybrid-potato}

Genomic prediction has already accelerated genetic progress in multiple
hybrid crops \autocite{Labroo2021}. \textbf{Chapter's 3, 4, and 5} have
several implications for genomic prediction applications in potato as a
hybrid crop. Considering factors related to parameterization of genetic
model or type of modelling framework (what I will call more generally,
\emph{model choice}), it is not the \emph{decisive} factor in the
outcome of genomic prediction strategy. Having reviewed extensions of
the traditional GBLUP, various shrinkage-based estimation methods, and
one implementation of the Gaussian kernel, similar performance was
observed with only marginal improvement between models. This is in
keeping with many similar studies in other row crops as well as observed
in tetraploid potato
\autocite{Sverrisdottir2017,Endelman2018,Amadeu2019,Wilson2021a,Schrauf2021}.
Whether the model is an extension of GBLUP or a new member of the
Bayesian alphabet, there is no clear or distinct advantage observed for
most traits.

As documented in other crops, often more important than the specific
genetic parameterisations of a model is the actual composition of its
training set \autocite{Dias2019}. In many breeding programs, training
sets often arise from the breeding material itself utilizing previous
breeding cycles to estimate the population genetic covariance or marker
effects for a genomic model. Within hybrid breeding schemas, training
sets are frequently developed from test cross blocks which often have
some factorial structure (e.g.~\textbf{t} testers by \textbf{x} new
candidates) or sparse modification \autocite{Jarquin2020}. These are
used both for the selection of novel lines within pools as well as for
the prediction of hybrid crosses. \textbf{Chapter's 3, 4, and 5} all
utilized four field trials which utilized just under 800 F1 hybrids
which were the progeny of 456 inbred parents in a sparse mating design.
While it could be demonstrated in an earlier chapter that the genetic
models based upon these training sets were technically
\emph{identifiable} \autocite{Xenakis2019}, the sparsity of the crossing
block likely led to bias in the estimation of our genetic variances and
of any predictions of the genetic effects \autocite{Mohring2011}.
Despite this, our training set still enabled genomic prediction of
hybrid performance and can likely be improved with further optimization.
Aside from incorporating future cycles to augment the training set,
there are many other methods relevant to increasing the phenotypic and
genetic variance in a training set
\autocite{Isidro2015,Berro2019,Ou2019}.

Related to training set composition is the topic of molecular marker
information density. These questions are often pursued along the lines
of predictive benefits in genomic modelling, the hope being that all
genetic variation can be interrogated by some perfect markerset. While
this cannot be fully achieved \autocite{Brachi2011}, markerset
composition is a worthwhile question in efficient selection
applications. In \textbf{Chapter's 4}, I considered a procedure for
comparing classical marker-assisted selection over genomic prediction
for several quantitative traits. One of the primary conclusions of this
research was that there are multiple complex traits in potato which
could be affectively selected for with only a handful of molecular
markers. This schema was relatively simple and could be augmented
through a more robust simulation of costs, genotype-by-environment
effects, and trial design \autocite{Peixoto2024}. A point should also be
made regarding our higher marker densities, that being, these were still
quite low. Many predictive modelling studies have used marker densities
in the magnitude of hundreds of thousands or million of molecular
markers \autocite{Druet2014,Weber2024}. Having said this, even
tetraploid studies have shown that small marker panels are still capable
of yielding acceptable results in genomic prediction and other trait
discovery applications which is in keeping with what I found in my
experimental chapters \autocite{Aalborg2024,Leyva-Perez2022}.

Aside from marker density, molecular marker content has captured the
attention of many geneticists. Whether it be the inclusion of functional
genetic information, enrichment of major QTL, or multi-omic data, all
seek to include biologically relevant information to bolster genetic
applications \autocite{Guo2016,MacLeod2016,Arouisse2024}. In this vein,
\textbf{Chapter 5} attempted genomic prediction based upon multiallelic
predictors through the inclusion of haplotag and IBD profiles.
Strategies for the development of multiallelic probe design and genetic
models have become a topic of interest in potato as an answer to
potato's genomic diversity \autocite{Tang2022}. While recent pan-genome
studies have shown less haplotype diversity among released potato
varieties then expected (particularly among European cultivars),
potential applications for multiallelic molecular data are still to be
identified \autocite{Sun2025}. Statistical models developed with the
haplotag probeset showed nearly identical performance with models using
traditional SNP-based predictors. These results would suggest little
benefit in the inclusion of multiallelic predictors for genome-wide
prediction based upon my survey of models used here. The performance of
the IBD models were notably lower than expectation, but as noted
already, this could be the product of high uncertainty for a subset of
ancestral founders. Whether or not IBD information can extend genomic
prediction applications, it is worthwhile to reiterate that founder
composition in a breeding program can be utilized in pre-breeding
efforts, understanding selection outcomes, and genetic diversity
management.

\section{Future research}\label{sec-future}

There are many facets of potato breeding that have not been sufficiently
addressed by current quantitative genetic frameworks. Here future
applications of quantitative genetics are considered with particular
emphasis on their role in improved fertility, pre-breeding methodology,
and breeding risk assessment for more robust potato breeding.

\subsection{Fertility Mechanisms}\label{fertility-mechanisms}

Fertility is a crucial factor for any seed-based crop. The success of
hybrid breeding systems are particularly dependent on seed cost price
which is chiefly the product of affordable and reliable seed production
systems \autocite{Mao1998,Longin2014}. Most research in potato fertility
up until this time has \emph{rightly} been focussed on large effect loci
like \emph{Sli} or important genes in wider fertility modules like
\emph{StCDF1} \autocite{Clot2020,Eggers2021,Song2022}. As these loci
continue to be used and fixed in breeding populations, an important
future step will be to survey a broader array of relevant traits in
diploid potato and assess them beyond QTL \autocite{Kempe2011}. The
typical targets for fertility in hybrid crops are pollen shed,
sufficient pollen viability, and synced male and female flower opening,
such that seed production is unencumbered
\autocite{Ortiz1997,Longin2012}. In conjunction with these traits, a
viable cytoplasmic male sterility (CMS) system is also crucial for F1
hybrid seed production. Not only does this aid in circumventing female
line emasculation costs \& minimizing F1 seed contamination, but also
reduces berry set in the F1 field crop thus eliminating seed-borne
volunteering post harvest. Recently, a CMS system has been proposed for
hybrid potato using the \emph{antherless} locus, \emph{Al}, on
chromosome 6 with male sterility expressed in germplasm containing the P
cytoplasm \autocite{Eggers2024}. Hybrids with P cytoplasm and homozygous
at the \emph{antherless} with \emph{alal} showed significantly lower
rates of berry set and less seed yields relative to fertile hybrids. One
operational concern with such a strategy is that it requires not only a
single locus to be fixed in both maternal and paternal lines, but all
maternal lines must also contain the P cytoplasm. It should also be
noted that while there is not a clear elucidation of cytoplasm's exact
role on agronomic and economic traits, there is evidence of cytoplasm
affecting maturity, starch content, and late blight
\autocite{Sanetomo2015}. This would have serious ramifications if
transferring a P cytoplasm to a breeding program's elite maternal lines
has an associated \emph{plastid-drag} synonymous to the use of QTLs
contained in large introgression regions. Further validation of the CMS
model can help assess cytoplasm substitution's risk on other economic
traits in hybrid potato.

\subsection{Pre-breeding for Quantitative
Improvement}\label{pre-breeding-for-quantitative-improvement}

The role of pre-breeding is often neglected with regard to complex trait
improvement of potato. Pre-breeding strategies in diploid potato are
particularly challenging as you are frequently interested in novel
material with a higher ploidy. Pre-breeding has at least three problems:
(1) How new germplasm should be screened, (2) How should ploidy
reduction be done, and (3) How should that material be incorporated into
a breeding program. None of these questions are very straightforward
with regard to a diploid potato breeding program and is decorated by
multiple hazards. In a more familiar crop breeding scenario, such as
maize, if a new program is being initiated, evaluating a base population
of landraces before introgressing into your breeding program is a
sensible strategy for trait improvement \autocite{Gorjanc2016}. However,
in diploid populations, novel tetraploid germplasm must first be
subjected to angiogenesis or \emph{prickle pollination} before
dihaploids can be introduced into a program
\autocite{Uijtewaal1987,Tai2003}. This process not only disrupts the
original trait architecture of that original tetraploid donor, but these
dihaploids frequently bear little to no resemblance to the original
donor making their original evaluation dubious.

Multiple screening approaches have been suggested, all of which, depend
on molecular marker information of some form. One approach proposed in
banana and potato suggests an extension of GBLUP whereby multi-ploidy
training sets could enable prediction from the 4x to 2x ploidy levels
\autocite{Nyine2018,Wilson2021}. The prediction accuracy observed in
these studies appear modest, but whether the endeavour of dedicated
tetraploid field evaluation for an augmented across-ploidy training set
is altogether justified is unclear. To avoid this, the IBD estimation
methodology from \textbf{chapter 5} could be considered as a potential
methodology for \emph{linking} the genetic value of dihaploids to their
tetraploid forbearer's. IBD estimation of deep pedigrees in diploids has
been realized in both simulated populations as well as real
\autocite{Li2021}. Additionally, IBD estimation has been conducted in
shallow structured populations in tetraploids
\autocite{Amadeu2021,Song2023}. Very little methodological development
would be needed to extend the IBD tracing from tetraploid to related
dihaploids enabling the estimation of allelic effects (in diploid
populations) from specific tetraploid haplotypes. If a novel donor has
already been introgressed into a program, IBD profiles can leveraged and
even extended to related tetraploids.

Both these methods essentially seek to explain the performance of
diploids by some molecular proxy in the tetraploid either through a
genetic covariance structure or via explicit IBD probabilities. One last
consideration for future improvements in pre-breeding is captured in
what is often called the next stage of breeding or \emph{``Breeding
4.0''} \autocite{Wallace2018}. The central premise is that a crop's
\emph{ideotype} can be designed from the ground up based upon a library
of known functionally annotated haplotypes which are then combined into
an optimal cultivar via cis or transgenic technologies
\autocite{Varshney2021}. Breeding is no longer a practice of finding the
proverbial needle in the haystack, but is instead the direct forging of
a variety from constituent components. This would make the role of
pre-breeding primarily focused on identifying those components. Recent
endeavours in diploid potato breeding could be described as examples of
such crop design, most notably, \textcite{Zhang2021}. Populations are
screened on the basis of functional genetic load based upon some
algorithmic method (e.g.~SIFT), and are then used to hasten the process
of inbred production. Whether this approach can produce a sufficient
volume of inbreds to initialize a breeding program is to be seen, but is
a tool which is already being used with the intent to design future
potato cultivars.

\subsection{Formalization of Risk}\label{formalization-of-risk}

Often, while considering a novel technology or more efficient breeding
schema, the first question asked is \emph{``What kind of genetic gains
can be expected with this implementation''}? This in of itself is not
unsound, but without asking \emph{``what are the risks of such an
implementation''} biometricians run the risk of overly optimistic
forecasting. This addendum is especially pertinent for potato breeders
due to many of the aforementioned challenges, chief among them,
environmental sensitivity \autocite{George2017}. Formalizing the
estimation of explicit risk of a breeding strategy is the province of
quantitative genetics and thus deserves greater attention. For example,
decreasing the cycle length of a schema by reducing the number of years
of trial evaluation is a classic route used for increasing genetic gain.
However, this also stands to increase the uncertainty around a candidate
if they have been exposed to an insufficient number of environments.
Embracing other tools might also bring risk to the forefront. Factor
analytic and random regression models have been lauded for their ability
to assess genotype stability and adaptability, and offer incredible
potential for evaluation in potato \autocite{Souza2020,Smith2018}.
Looking to methods utilized outside of plant breeding such as the use of
hazard models could also be useful for assessing the long-term stability
of a potential potato candidate \autocite{Yazdi2002a}.

\subsection{Conclusion}\label{conclusion}

I present here a first examination into the quantitative nature of
several complex traits in hybrid potato. Additionally, through the
survey of a variety of statistical models, I offer a framework for
evaluating hybrid potato genetic architecture, genotye-by-environment
interaction, and genetic progress. This thesis has also demonstrated the
transformative potential of biometrical methods in advancing hybrid
potato breeding. Our findings underscore the importance of building on
traditional statistical formulations for both inferential and predictive
applications in potato breeding. These insights not only bridge critical
gaps in potato breeding but also align with broader trends in crop
improvement, where quantitative genetics and genomic technologies are
reshaping traditional paradigms. Moving forward, the integration of
these methods into breeding programs promises to enhance precision,
reduce cycle times, and unlock the full potential of hybrid potato as a
sustainable crop for future agriculture.


\printbibliography[title=References]



\end{document}
