---
title: "Discussion"
output-file: "06_discussion"
---

## Introduction

Hastened genetic progress in potato is dependent on the continued application of quantitative genetics methods in breeding programs. Like many vegetable crops, the application of statistical genetic theory has only received significant attention the past two decades in potato. Before this, complex trait improvement in conventional potato breeding was reliant on phenotypic selection and marked by several inefficiencies  including underpowered field evaluations and long release cycles. With the wider application of successful technologies that have had positive success in other field crops, higher success of potato improvement is possible. These include both methods such as variance component estimation, genomic prediction, selection index theory as well as the importing of other breeding schemas like recurrent selection or line breeding.

In this thesis, we consider the conversion of clonal potato to a diploid hybrid crop as an exercise of applied biometrical methods. To our knowledge, this thesis is the first to interrogate the genetic architecture of a hybrid potato population and assess both features in these populations and the adequacy of specific statistical methods applied to them. This was the form our questions took throughout this thesis. We first examined the genetical properties of the F1 hybrid seedling trials including genetic correlations between multiple tuber yield components (chapter 2; @Adams2022). We also considered the population structure of these hybrids with the introduction of inbred molecular marker information (chapter 3; @Adams2023) as well as with ancestral founder profiles (chapter 5). After having estimated these genetic parameters, we tested multiple predictive modelling methodologies testing the marginal gains of predicting dominance effects (chapter 3; @Adams2023), potential benefits of marker assisted selection for complex traits (chapter 4), and various multiallelic parameterisations for genome-wide regression (chapter 5). These topics cover an important range of topics for modern selection methods in a potato breeding program.

Here we provide a synthesis of the previous four experimental chapters with two major aims. We first wish to identify the most important findings of this research, and second, outline the direct implications of our research on potato more generally. We then conclude this thesis with reflections on future research.

## Biometrical analysis of field trials

A core activity in breeding programs is the regular evaluation of new parental combinations to screen for genetically improved offspring. For this reason, within-trial and multi-environmental trial design is so heavily emphasised for sufficient statistical power and optimal resource allocation [@]. This permits proper extraction of spatial and other non-genetic components in a field trial and unbiased evaluation of genotype performance. From this successful spatial analysis, we are then able to measure the genetic parameters of interest whether it be comparison of genotype means, or the estimation of genetic correlations between traits.

In conventional potato breeding, formal trial analysis is traditionally reserved in the C1 or C2 generations when there is a sufficient number of clones for replication [@Gopal2015; @Paget2017]. This is in contrast to other field crops where true seed is the predominant propagule and usually allows for suitable replication after the initial crossing depending on the stage of the candidates [@Zystro2018]. An overlooked property of hybrid breeding in potato is the necessity of true seed and the ability to conduct uniform trialling on seed sown or transplants with multi-plant plots rather than single plant hills. This design decision of course carries with it consequences, chief among them, being poor correlation in multiple tuber traits between the seedling S1 and clonal generations [@Anderson1981]. This is addressed in full in @sec-implications, but suffice it to say that the field evaluation of hybrid potato in the S1 generation provides opportunities for explicit field design with sufficiently high precision for multiple tuber phenotypes [@VanDijk2021a].

In **chapter 2**, such an exercise was conducted in the form of a multi-trial analysis on several hundred F1 hybrids. A penalized splines based procedure was used for spatially de-trending our hybrid observations for three tuber phenotypes. These de-trended phenotypes were then placed in a multivariate linear mixed model for the estimation of the genetic (co-)variances for the GCA, SCA, and several interactive effects. In **chapter 3**, this was followed up with a traditional linear mixed modelling procedure where multiple row column designs with various residual structures were tested on an augmented dataset [@Gilmour1997]. The BLUEs and standard errors from these models where then used in a full genomic model where genomically estimated BLUPs were evaluated in a cross-validation schema. Both modelling procedures from these chapters follow the general form of a two-stage mixed model analysis where trials are independently analyzed followed by an combined analysis on the genotypic means (most commonly) together with weights on those genotypic observations [@Piepho2012]. Its generally known that two-stage approaches tend to be less accurate than than their single stage counterparts especially with greater model complexity [@Gogel2018]. All this being said, single-stage approaches tend to be _unwieldy_ in their specification and computation and the consequences of a two-stage approach are unlikely to change any inferential conclusions from these two chapters [@Damesa2017].

A central activity in hybrid breeding is assessing the genetic potential of a novel parental line. 

The results of **chapter 2** showed that the predicted GCAs sufficiently captured . This not only confirms the utility of GCA estimation for summarising parental performance in hybrid potato.

adequately represented by their parental combinations ( chapter 2). 
- This is not something unique to hybrids, as there are multiple studies in tetraploid potato that make use of the
- A core concept in hybrid breeding is the ability to assess the potential of hybrid crosses on the basis of parental value.
- This forms the primary basis of genetic improvement in a hybrid breeding program.
- Our statistical formulation allowed us to present genetic models which produce those breeding values and assess their capture on phenotypes of interest.

From **chapter 2**, some insight around the genetic architecture of yield components was gleaned in the form of genetic correlations. Not only were we able to extract the covariance parameters for overall genetic covariance, but also express this per GCA and SCA variance. This is worth re-emphasis for the following reasons. First, potato is a crop has a relatively large trait surface comprising tens of traits of interest [@Gebhardt2013], many of them being complex with low heritabilities and non-independent relationships. Because of this there is a real need to better utilize multivariate methods in genetic improvement in potato. In addition, hybrid breeding schemas are also augmented by multivariate methods whether based upon phenotype alone or also for multivariate genomic prediction applications [@Wang2017]. To lend some credence to multivariate applications in potato, we can consider the genetic covariances for GCA's from Figure \ref{fig:gca-coef-full-pairs} together with their full phenotypic variance matrix. Assuming the GCA variances estimated here would be roughly equivalent to those estimated from a test cross schema in a hybrid breeding program, we could estimate the following selection response using the multivariate breeder's equation as expressed by @Lande1983. If we performed truncation selection on tuber volume (s = 2 $cm^3$) and conducted random intermating between candidates, then the expected selection response can be estimated:

```{R get-covariances}
#| echo: false
#| output: asis
library(data.table)
library(glue)
#Get your variance components from the first chapter
vrs <- fread("./assets/VarianceComponentsFullModel.csv")
#extract trait 1 and trait 2 and component
tr <- c("TV", "TY", "TN")
#init matrix
m <- matrix(NA, nrow = 3, ncol = 3, dimnames = list(tr, tr))

#filter and prepare your variances
setnames(vrs, 1, "effect")
vrs <- vrs[effect != "units:trait!R"]
rgx <- "^(.*)!trait_(.*):(.*)$"
vrs[, c("eff", "tr1", "tr2") := tstrsplit(gsub(rgx, "\\1 \\2 \\3", effect), " ")
    ][, c("effect", "std.error", "z.ratio", "bound", "%ch") := NULL]
#Get the opposite side of diagonal
full <- rbind(vrs, vrs[tr1 != tr2, .(component, eff, tr1 = tr2, tr2 = tr1)])
#last but not least but scale the additive effects correctly
full[eff == "trait:dam", component := component * 2]

#Get your matrices
vList <- lapply(split(full, full$eff), \(f) {
  m[f[, cbind(tr1, tr2)]] <- f$component
  m
})
P <- Reduce(`+`, vList)
G <- vList$`trait:dam`
Pinv <- solve(P)
S <- c(2, 0, 0)
R <- G %*% Pinv %*% S

# do ungodly things to get your latex
#Get g text
Gtext <- paste0(apply(round(G, 2), 1, paste0, collapse = " & "),
                collapse = " \\\\ ")
#then P
Pinvtext <- paste0(apply(round(Pinv, 2), 1, paste0, collapse = " & "),
                   collapse = " \\\\ ")
#and S
stext <- "\\begin{bmatrix} 2 \\\\ 0 \\\\ 0 \\end{bmatrix}"
#and finally R
Rtext <- paste0(apply(round(R, 2), 1, paste0, collapse = " & "),
                collapse = " \\\\ ")
#hella messy
lat <- glue("$$ \\mathrm {{R = G \\cdot P^{{-1}} \\cdot S}}$$
  $$ \\mathrm {{R}} = \\begin{{bmatrix}}{Gtext}\\end{{bmatrix}}\\cdot\\begin{{bmatrix}}{Pinvtext}\\end{{bmatrix}} \\cdot {stext} $$
  $$\\mathrm {{R}} = \\begin{{bmatrix}}{Rtext}\\end{{bmatrix}}$$")
lat
```

This would suggest a mean increase of about 1.5 $cm^3$ in tuber volume with a minor increase in total tuber yield and little change in tuber number. This of course could be taken a step further and transformed into an index using a variety of methods [@Kempthorne1959; @Bulmer1981]. More attention is needed here to make these methods more scalable in applied settings. Traditional linear mixed modelling tends to be inpractical with more than 10 variates making other methods such as Bayesian learning more attractive [@Blows2009; @Runcie2013].

## Genomic prediction in hybrid potato

  - These same models can be overlaid with marker data for genomic prediction/selection applications. (chapter 3)
  - The ability to apply genomic prediction in a crop represents a major milestone in hybrid breeding. Because hybrid breeding tends to have longer release cycles, genomic prediction is necessary in hastening informed selection decisions.
    v) Multiple molecular selection strategies were viable for several traits (chapter 4).
    vi) Potential for multiallelic marker information in predictive modelling (chapter 5).
         A. We examined multiple forms of multiallelic marker information with a focus on differences between IBS and IBD based loci.

## Methods of selection

- Breeding programs have undergone substantial transformations with the introduction of alternative methods of selection. 

## Wider Implications

### Focus on GCA in hybrid breeding

 - SCA is a genetic residual [@Bernardo2016] (chapter 2)
         A.  Not something worth trying to predict in diploids.
         B.  Using GCA's for selection in young potato populations.
         C. [@Labroo2023]

### Multivariate applications for potato breeders. {#sec-implications}

Future applications of statistical models (3 - 4 paragraphs)
    i) Multi-trait modelling to estimate genetic correlation between tuber generations
         A.  [@Gopal1998]: GCA does not appear to be very repeatable in early generations, suggesting that even family-based analyses do not give stable components across generations.
         B.  [@Davies1974]: Also examines the lack of suitability in selection in 1-hill stage
         C. [@Maris1988]: Excellent paper looking at the lack of correlation between seedling, first, and second clonal generations.

         A.  Index selection can be of direct use to breeders.
         B.  Technical problems applying this in production.

### Successful genomic prediction in hybrid potato

Model choice is useful but not the most important factor for selection.
As documented in other crops, often more important than the minutia of specific genetic parameterisations is the actual composition of a training set [@Dias2019]. 
An additional point can also be made regarding molecular marker information. A recurrent topic in our field is the importance of marker density and adequately tagging every linkage block in segregating regions in a population. For our study we used a probeset with a relatively low genome coverage and still found remarkable success according to our cross-validation schema.
    iv) Technologies like genomic prediction can be applied quite simply
         A.  Require smaller training set sizes relative to tetraploids
         A.
 iii) Genomic prediction as a vehicle for population change
         A.  You are driving selection of parent development
         B.  Hybrid prediction for coming cycles [@Zhao2015b].
   vii) Multiallelism likely a benefit for major QTL, and little else

    v) Crossing schema for training set development (chapter 3).

## Future research

There are many areas in the world of potato breeding which could be improved with further biometrical applications. We consider here three areas of study which deserve more careful attention in terms of guaranteeing the success of potato breeding.

![(left) The classic breeder's equation where genetic gain is a linear function of selection intensity, selection accuracy, and the genetic standard deviation divided by the length of cycle and breeder's risk (right) which is the product of selection intensity, the compliment of selection accuracy, and the genetic by environment standard deviation divided by the number of trials.](./assets/selection_risk.pdf){#fig-risk width=80%}

Sufficient fertility is a crucial trait for any seed crop. This is, however, conditioned on enough vigour in fertility properties (e.g. excellent pollen viability, synced male and female flower opening times) among parental lines such that seed production is unencumbered [@Longin2012].

A currently under-served corner in the genetic improvement of potato is the identification of novel germplasm for complex trait improvement. The specific challenge of identifying superior candidates among tetraploids for diploid potato programs is laced with multiple hazards. First, the genetic architectures of diploid and tetraploids are not equivalent, or at the very least, attempts at genomic modelling containing both polyploid and diploid potato have not supported this [@Wilson2021].     In **chapter 5**, we considered 
    ii) IBD founder tracing in pre-breeding generations in potato
         A.  Link to Genome Design [@Zhang2021].
    

Often, while considering a novel technology or more efficient breeding schema, the first question asked is _"What kind of genetic gains can be expected with this implementation"_? This in of itself is not unsound, but without asking _"what are the risks of such an implementation"_ biometricians run the risk of overly optimistic forecasting. This addendum is especially pertinent for potato breeders due to many of the aforementioned challenges, chief among them, environmental sensitivity [@George2017]. Formalizing the estimation of explicit risk of a breeding strategy is the province of quantitative genetics and this deserves emphasis. To this end, we can conceptualize an analogue to the classic breeder's equation where the potential risk tradeoffs associated with such a strategy are manifest [@fig-risk]. For example, decreasing the cycle length of a schema by reducing the number of years of trial evaluation can technically hasten genetic gain. Naturally, however, this would also increase the risk or uncertainty around these candidates if you have exposed them to an insufficient number of environments. Embracing other tools might also bring risk to the forefront; methods like random regression or borrowing hazard models from the animal breeding world could be useful for assessing the stability of a potential candidate [@Yazdi2002a; @Souza2020].


5. Wrapping up / Conclusions about hybrid breeding in potato  (2 paragraphs)
   i) Current status of statistical methods in potato
   ii) This thesis' place in advancing knowledge about hybrid potato

::: {.content-hidden}
    i) Hybrid breeding requires statistical modelling to evaluate genetic variance and trait architecture (chapter 2)
         A.  Can use this for future selection forecasting
         B.  Impact of selection on other traits
        iii) We can evaluate the efficiency of different technologies 
    iv) We can evaluate the efficiency of different models and information

3. Statistical genetic topics critical in potato
    i) Dealing with low seedling and tuber-sown genetic correlations
         A.  Touch on seedling versus clonal cropping systems 
         B.  Review literature on lack of correlation 
         C.  Propose early seedling evaluation and multi-trait prediction models as potential solution
    ii) Evaluating GxE and sensitivity rigorously
    iii) Germplasm aquisition and evaluation (pre-breeding topics)
         A.  Siezing ploidy. Effective tetraploid mining for diploid breeding
         B.  Address other breeding strategies such as bridge breeding (Corentin Clot)

4. Statistical genetic topics crucial in hybrid breeding 
    i) Fertility and seed production in potato inbreds
         A.  Affordable production
         B.  inbreeding depression
         C.  Genetic factors outside _sli_
    ii) Genetic transformation
         A.  The collaborative role of gene-editting in quantitative trait improvement
         B.  The need for regeneration and transformation as traits 
              - Necessary for Doubled haploids, genetic transformation, and double monoploid production 
              - Genetic variation in response identified in potato 
              - Genes found in other crops [@Koornneef1993]
         C.  Building elite inducers [@Delzer2024]
    iii) Pipeline for new traits for new production systems



## QTL: "_I know it when I see it_"

* It is widely understood that QTL mapping is only useful for turning up large variants and they in themselves cannot reconstitute all of the relevant heritable variation for a trait. Echoing other geneticists remarks on QTL mapping, _"all that glitters is not gold"_. However, as seen in these results they are not without some utility. They can be used for relatively accurate selection assuming that target populations are large enough.


* There are many reasons why a breeder might want to understand the nature of major variants in their breeding program.

* However, unless this is something that either disrupts normal biological processes (severe gametic selection), this likely doesn't need to be 

This can best be segmented into the categories of _nice to know_ and _need to know_. 

## Multiallelism: An unnecessary consideration?

- Breeding is more than just prediction. The use of information by a breeder is not confined to breeding value estimation. 

## Methods should provide insights

Very serious discussion about things.

:::
