\section{Supplementary Material}

\subsection{Founder Origin Sparsity}\label{founder-origin-sparsity}

After the conclusion of the RABBIT pipeline, founder origin sparsity was
assessed genome-wide using a rolling-window over average number of sites
which had no confident haplotype call (probability under 0.5). This
showed a high degree of missing haplotype calls on chromosome's 1 and 5
(Sup Fig~\ref{fig-sparse}).

\begin{figure}[h!]

\centering{

\includegraphics{figs_05/fig-sparse-1.pdf}

}

\caption{\label{fig-sparse}Smoothed sparsity of founder-origin alleles
for chromosome's 1 and 5.}

\end{figure}%

\subsection{Multiallelic Marker Matrix
Parameterization}\label{multiallelic-marker-matrix-parameterization}

For both haplotag and IBD based markersets, let us assign each allele
some identifier \{a, b, c\} to denote genotypic states with the former
representing a haplotype sequence of 75 basepairs in length and the
latter being an assigned founder origin. We then consider two loci (l1
and l2) with the reference allele (\(a\)) being either the reference
haplotag allele or the most prevalent founder allele in the population.
The multiallelic incidence matrix is centred using the cross product of
a column vector of 1's
\(\left ( \mathbf{1}^T_4 = \left ( 1, 1, 1, 1 \right )^T \right )\) and
a vector of allelic frequencies excluding the reference allele
\(\left ( \mathbf{p} = \begin{bmatrix} l1b & l1c & l2b \\ \frac{3}{8} & \frac{1}{4} & \frac{1}{2}\end{bmatrix} \right )\).

\[
\begin{aligned}
\kbordermatrix{
    & l1 & l2 \\
  g1 & aa & ab  \\
  g2 & cc & bb  \\
  g3 & bb & aa  \\
  g4 & ab & ab   
} 
\Rightarrow
\kbordermatrix{
    & l1a & l1b & l1c & l2a & l2b \\
  g1 & 2 & 0 & 0 & 1 & 1 \\
  g2 & 0 & 0 & 2 & 0 & 2 \\
  g3 & 0 & 2 & 0 & 2 & 0 \\
  g4 & 1 & 1 & 0 & 1 & 1
}
\Rightarrow
\kbordermatrix{
    & l1b & l1c & l2b \\
  g1 & 0 & 0  & 1 \\
  g2 & 0 & 2  & 2 \\
  g3 & 2 & 0  & 0 \\
  g4 & 1 & 0  & 1 
}
\Rightarrow 
\\
\mathbf{1}^T_4 \bigotimes 2\mathbf{p} - \kbordermatrix{
    & l1b & l1c & l2b \\
  g1 & 0 & 0  & 1 \\
  g2 & 0 & 2  & 2 \\
  g3 & 2 & 0  & 0 \\
  g4 & 1 & 0  & 1 
}
\Rightarrow
\kbordermatrix{
    & l1b & l1c & l2b \\
  g1 & 2p_{l1b} & 2p_{l1c} & 2p_{l2b} - 1 \\
  g2 & 2p_{l1b} & 2p_{l1c} - 2  & 2p_{l2b} - 2 \\
  g3 & 2p_{l1b} - 2 & 2p_{l1c} & 2p_{l2b} \\
  g4 & 2p_{l1b} - 1 & 2p_{l1c}  & 2p_{l2b} - 1 
}
\Rightarrow
\kbordermatrix{
    & l1b & l1c & l2b \\
  g1 & \frac{3}{4} & \frac{1}{2} &  0 \\
  g2 & \frac{3}{4} & -\frac{3}{2}  & -1 \\
  g3 & -\frac{5}{4} & \frac{1}{2}  & 1 \\
  g4 & -\frac{1}{4}  & \frac{1}{2}  & 0 
}
\end{aligned}
\]

This centred matrix of either IBD founder or haplotag alleles was then
used in subsequent genome-wide regression models.


\subsection{LASSO and GROUP LASSO}\label{lasso-and-group-lasso}

For the LASSO (LS) training runs, the degrees of freedom and estimated
\(\lambda\) parameter were recorded. This is shown for all traits in
(Sup Fig~\ref{fig-lasso}). For the grouped LASSO (GLS), the predictive
accuracy of that training run is also captured and shown only for the
IBD probeset in (Sup Fig~\ref{fig-lasso-2}).

\begin{figure}[h!]

\centering{

\includegraphics{figs_05/fig-lasso-1.pdf}

}

\caption{\label{fig-lasso}The estimated \(\lambda\) which the associated
degrees of freedom for each predictor set and trait for all lasso
models.}

\end{figure}%

\begin{figure}[h!]

\centering{

\includegraphics{figs_05/fig-lasso-2-1.pdf}

}

\caption{\label{fig-lasso-2}The estimated \(\lambda\) plotted against
the model prediction accuracy with the associated degrees of freedom for
the multiallelic predictor sets for traditional (a) and (b) grouped
lasso models using IBD-based predictors.}

\end{figure}%

\subsection{Sampled Posterior Distributions for Gaussian Kernel
parameters}\label{sampled-posterior-distributions-for-gaussian-kernel-parameters}

The Markov Chain Monte Carlo estimation of our parameters for the
Transformed Gaussian Kernel are shown over all runs in (Sup
Fig~\ref{fig-gk-1}) and (Sup Fig~\ref{fig-gk-2}). These represent the
sampled joint posterior distributions of the bandwidth and form
parameters (\((\hat{h},~\hat{\phi})\)) and the estimated genetic and
residual variances
(\((\hat{\sigma}_g\),\textasciitilde{}\(\hat{\sigma}_e)\)),
respectively.

\begin{figure}[h!]

\centering{

\includegraphics{figs_05/fig-gk-1-1.pdf}

}

\caption{\label{fig-gk-1}Bivariate heat map of the posterior mode of
\(\hat{h}\) and \(\hat{\phi}\) from each fold of the cross-validation
for each trait.}

\end{figure}%

\begin{figure}[h!]

\centering{

\includegraphics{figs_05/fig-gk-2-1.pdf}

}

\caption{\label{fig-gk-2}Bivariate heat map of the posterior mode of
\(\hat{\sigma_g}\) and \(\hat{\sigma_e}\) from each fold of the
cross-validation for each trait.}

\end{figure}

